apps/ledger-live-desktop/src/storyly/useStoryly.tsx

```diff
@@ -53,13 +52,15 @@ export const useStoryly = (
       dispatch(closeAllModal());
       setDrawer();
     });
+
+    ref.current?.setLang({ language });
   });
```

Comment:
Same question here. But also I know it's not from this PR but why is there no dependencies here ?

Reply:
cannot answer, historically it have been done, i can't analyze the impact on the app